db/migration/1676376942081-links.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class links1676376942081 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        queryRunner.query(`
+        CREATE TABLE links (
```

Comment:
I think this table should not be called just links. How about posts_gdocs_links?